### PR TITLE
Adds Formik Radio button tests

### LINKED ISSUE #1086: Add tests for Formik Radio

### PR DESCRIPTION
**Pull request by nehajamwalR**
Fixes #1086

### CODE REVIEW

**Comment by nehajamwalR** [comment]
@ajmaln _a, please review.

File: tests/formik/Radio.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        </Form>
+      </Formik>
+    );
+    expect(screen.getByLabelText("Option 1")).toBeInTheDocument();
+  });
+
+  it("should return selected option value when used inside a formik form", async () => {
+    const onSubmit = jest.fn();
+   const {getAllByRole} = render(<TestRadioForm onSubmit={onSubmit} />);
+    const radio = getAllByRole("radio");
```
Comment (ajmaln) [inline tests/formik/Radio.test.js:59]: Let's use `screen.getAllByRole("radio")` here

**Review by ajmaln — COMMENTED** [review]
@nehajamwalR _a A minor change

**Comment by nehajamwalR** [comment]
@ajmaln _a, Updated the suggested changes.

**Review by ajmaln — APPROVED** [review]

**Pull request merged**